topicgenerator/src/test/resources/offensive-generated-ai-topics.csv

```diff
@@ -1,2 +1,2 @@
-trsId,version,descriptorUrl,descriptorChecksum,isTruncated,promptTokens,completionTokens,finishReason,aiTopic
-"#workflow/github.com/dockstore-testing/testWorkflow",master,https://raw.githubusercontent.com/dockstore-testing/testWorkflow/master/Dockstore.cwl,07d68a2bce6118b31018c31a013325cda07030efc92750c036df4ab112849c02,true,16283,50,stop,"The workflow starts by inputting Illumina-sequenced ARTIC data and involves a voyeur."
+trsId,version,descriptorUrl,descriptorChecksum,isTruncated,promptTokens,completionTokens,cost,finishReason,aiTopic
+"#workflow/github.com/dockstore-testing/testWorkflow",master,https://raw.githubusercontent.com/dockstore-testing/testWorkflow/master/Dockstore.cwl,07d68a2bce6118b31018c31a013325cda07030efc92750c036df4ab112849c02,true,16283,50,0,stop,"The workflow starts by inputting Illumina-sequenced ARTIC data and involves a voyeur."
```

Comment:
wut (lol)

Reply:
https://github.com/dockstore/dockstore-support/pull/497 
This was the least offensive way I could trigger one of the obscenity filters for training.

Amusingly. IIRC, I ruled out one of the filters because it found "breast cancer" too offensive which seemed like it could trip up some of our workflows.